agent/kafka_handler.go

```diff
@@ -1,8 +1,18 @@
 package agent
 
 import (
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spirit-labs/tektite/common"
 	"github.com/spirit-labs/tektite/kafkaprotocol"
 	"github.com/spirit-labs/tektite/kafkaserver2"
+	"github.com/spirit-labs/tektite/topicmeta"
+)
+
+const (
+	defaultRetentionTime = 7 * 24 * time.Hour
```

Comment:
this should be configurable

Reply:
should we pass it as a param here?

```
func (a *Agent) newKafkaHandler(ctx kafkaserver2.ConnectionContext) kafkaprotocol.RequestHandler {
	return &kafkaHandler{
		agent: a,
		ctx:   ctx,
	}
}

type kafkaHandler struct {
	agent *Agent
	ctx   kafkaserver2.ConnectionContext
}
```

or maybe as part of some config?

Reply:
> or maybe as part of some config?

Yes, it should be configurable